### PR TITLE
[gui-tests] Unskip syncing tests

### LINKED ISSUE #9762: refactor tests in `tst_syncing` suite
A few test scenarios from the `tst_syncing` suite are failing as the step definitions are unable to find the `choose_what_to-sync` button in the new wizard.  The test was written for version 2.10, so now it is incompatible with version 3.0. It is because the wizard of client version 3.0 is somewhat different than version 2.10. In version 3.0, the `choose_what_to_sync` button appears only when we select the `Configure synchronization manually` option in the advanced section. So now we'll have to refactor the test accordingly.

Scenarios to be worked on in this issue:

- [x] Scenario: Sync all is selected by default
- [x] Scenario: Sync only one folder from the server
- [x] Scenario: Connect account with manual sync folder option   [Scenario removed in this PR](https://github.com/owncloud/client/pull/9847)
- [x] Scenario: sort folders list by name and size

### PR DESCRIPTION
**Pull request by saw-jan**
Refactored steps to use updated UI path and unskipped the syncing tests.

Following tests are unskipped:
- `Scenario: Syncing a file from the server and creating a conflict`
- `Scenario: sort folders list by name and size`

Related tests:
Closes #9762, #9680

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/13541/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/13541/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/13541/guiReportUpload/serverlog.log

Screenshots:
- [Adding_account_with_vfs_enabled.png](https://cache.owncloud.com/public/owncloud/client/13541/guiReportUpload/screenshots/Adding_account_with_vfs_enabled.png)
- [Sync_all_is_selected_by_default.png](https://cache.owncloud.com/public/owncloud/client/13541/guiReportUpload/screenshots/Sync_all_is_selected_by_default.png)
- [Sync_only_one_folder_from_the_server.png](https://cache.owncloud.com/public/owncloud/client/13541/guiReportUpload/screenshots/Sync_only_one_folder_from_the_server.png)
- [Try_to_enable_experimental_vfs_option_and_cancel_it.png](https://cache.owncloud.com/public/owncloud/client/13541/guiReportUpload/screenshots/Try_to_enable_experimental_vfs_option_and_cancel_it.png)
- [sort_folders_list_by_name_and_size.png](https://cache.owncloud.com/public/owncloud/client/13541/guiReportUpload/screenshots/sort_folders_list_by_name_and_size.png)

</details>

File: test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py [hunk 1]
```diff
@@ -148,7 +97,7 @@ def addUserCreds(self, context):
         )
         squish.clickButton(squish.waitForObject(self.NEXT_BUTTON))
 
-    def finishSetup(self):
+    def nextStep(self):
```
Comment (amrita-shrestha) [inline test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py:100]: why change to nextStep?

File: test/gui/shared/scripts/pageObjects/SyncConnectionWizard.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    )
+
+    def setSyncPathInSyncConnectionWizard(self, context):
+        squish.waitForObject(self.ADD_FOLDER_SYNC_CONNECTION_WIZARD)
+        squish.type(
+            self.CHOOSE_LOCAL_SYNC_FOLDER, context.userData['currentUserSyncPath']
+        )
+        self.nexStep()
+
+    def nexStep(self):
```
Comment (amrita-shrestha) [inline test/gui/shared/scripts/pageObjects/SyncConnectionWizard.py:47]: did you intentionally name it nexStep?

File: test/gui/tst_addAccount/test.feature [hunk 3]
```diff
@@ -47,24 +47,24 @@ Feature: adding accounts
...
         Given the user has started the client
-        When the user adds the first account with advanced configuration
+        And the user has added the following account information:
             | server   | %local_server% |
             | user     | Alice          |
             | password | 1234           |
-        And the user selects virtual_files option in advanced section
-        And the user selects enable_experimental_placeholder_mode option in enable experimental feature dialogue box
+        When the user selects vfs option in advanced section
+        And the user confirms the enable experimental vfs option
```
Comment (amrita-shrestha) [inline test/gui/tst_addAccount/test.feature:57]: I think select fit more than confirm

File: test/gui/tst_addAccount/test.feature [hunk 4]
```diff
@@ -47,24 +47,24 @@ Feature: adding accounts
...
         Given the user has started the client
-        When the user adds the first account with advanced configuration
+        And the user has added the following account information:
             | server   | %local_server% |
             | user     | Alice          |
             | password | 1234           |
-        And the user selects virtual_files option in advanced section
-        And the user selects stay_safe option in enable experimental feature dialogue box
+        When the user selects vfs option in advanced section
+        And the user cancels the enable experimental vfs option
```
Comment (amrita-shrestha) [inline test/gui/tst_addAccount/test.feature:68]: I think you change the whole scenario purpose

File: test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py (hunk 1 above)
Comment (saw-jan) [reply test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py:100]: this function moves to the next dailog
`Server-(nextStep)->Cred-(nextStep)`

File: test/gui/tst_addAccount/test.feature (hunk 3 above)
Comment (saw-jan) [reply test/gui/tst_addAccount/test.feature:57]: This step can have two variations (`confirms` and `cancels`)

File: test/gui/tst_addAccount/test.feature (hunk 4 above)
Comment (saw-jan) [reply test/gui/tst_addAccount/test.feature:68]: How? Isn't the scenario trying to enable vfs and then cancel it with the `stay_safe` option?
I read the previous scenario that way. Please, correct me if it isn't

Comment (amrita-shrestha) [reply test/gui/tst_addAccount/test.feature:68]: To my knowledge, it's not about canceling vfs option. It's about adding account with stay safe advance option

Comment (saw-jan) [reply test/gui/tst_addAccount/test.feature:68]: I believe `stay safe` means and does cancel the vfs enable action (i.e. revert to recommended option - sync everything)

**Review by amrita-shrestha — APPROVED** [review]
lgtm

**Comment by saw-jan** [comment]
[NOTE] I want to merge this first: https://github.com/owncloud/client/pull/10261

**Comment by SwikritiT** [comment]
@saw-jan https://github.com/owncloud/client/pull/10261 is merged now!

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests-ocis</strong> https://drone.owncloud.com/owncloud/client/13771/7/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/13771/ocis/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/13771/ocis/guiReportUpload/serverlog.log

Screenshots:
- [Syncing_a_file_from_the_server_and_creating_a_conflict.png](https://cache.owncloud.com/public/owncloud/client/13771/ocis/guiReportUpload/screenshots/Syncing_a_file_from_the_server_and_creating_a_conflict.png)

</details>

**Pull request merged**